webapp/src/app/pages/components/knowledge-graph/concept-map/concept-map.component.ts

```
@@ -715,9 +709,9 @@
       let accTab2 = document.getElementById('accordionTab2');
       if (accTab1 && accTab2) {
         let accordionTab1 = document.getElementById('accordionTab1')
-        .childNodes[0].childNodes[0].childNodes[0] as HTMLElement;
+          .childNodes[0].childNodes[0].childNodes[0] as HTMLElement;
         let accordionTab2 = document.getElementById('accordionTab2')
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'accordionTab2' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)